Block/BlockContextManagerInterface.php

```diff
@@ -39,6 +39,13 @@ public function addSettingsByType($type, array $settings, $replace = false);
     public function addSettingsByClass($class, array $settings, $replace = false);
 
     /**
+     * @param string $type    Block type to check for
+     *
+     * @return bool
+     */
+    public function exists($type);
```

Comment:
@mweimerskirch just remove the function from the interface from now. It can be added in the next major version.